roles/ceph-common/tasks/main.yml

```diff
@@ -49,6 +49,7 @@
     ceph_release: "{{ ceph_stable_release }}"
   when:
     - ceph_origin == 'repository'
+    - ceph_stable_release != 'dummy'
```

Comment:
According to your commit message, I think adding `ceph_repository != 'dev'` would be more accurate

Reply:
sure, done